src/development/tests.md

```diff
@@ -1,22 +1,23 @@
 # Implementing a Kernel Test
 
-This guide covers how to write in-kernel tests of hardware functionality.
-For example, if you have implemented a chip peripheral, you may want to write
+This guide covers how to write in-kernel tests of hardware functionality. For
+example, if you have implemented a chip peripheral, you may want to write
 in-kernel tests of that peripheral to test peripheral-specific functionality
-that will not be exposed via the HIL for that peripheral. This guide
-outlines the general steps for implementing kernel tests.
+that will not be exposed via the HIL for that peripheral. This guide outlines
+the general steps for implementing kernel tests.
 
 
 ## Setup
 
 This guide assumes you have existing chip, board, or architecture specific code
 that you wish to test from within the kernel.
 
-> Note: If you wish to test kernel code with no hardware dependencies at all, such as
-> a ring buffer implementation, you can use cargo's test framework instead. These
-> tests can be run by simply calling `cargo test` within the crate that the test
-> is located, and will be executed during CI for all tests merged into upstream Tock.
-> An example of this approach can be found in `kernel/src/common/ring_buffer.rs`.
+> Note: If you wish to test kernel code with no hardware dependencies at all,
+> such as a ring buffer implementation, you can use cargo's test framework
+> instead. These tests can be run by simply calling `cargo test` within the
+> crate that the test is located, and will be executed during CI for all tests
+> merged into upstream Tock. An example of this approach can be found in
+> `kernel/src/collections/ring_buffer.rs`.
```

Comment:
This is also looks like unchanged line-wrap churn?

Reply:
I don't understand why churn is bad. We like formatting, we use rusftfmt. Periodically we have to re-format our docs. It might be nice if those were separate PRs, but that doesn't happen in practice. It might also be nice to have mdfmt.